cms/src/components/forms/LSelect.vue

```diff
@@ -5,7 +5,7 @@ export default {
 </script>
 
 <script setup lang="ts">
-import { type Component, type StyleValue } from "vue";
+import { ref, watch, type Component, type StyleValue } from "vue";
```

Comment:
Getting a few linting errors here for imports that are not used

Reply:
Fixed.